### ISSUE TITLE
 [mdStepper] change md-elevation and the stepper doesn't change

### ISSUE THREAD

**Issue by ansonhorse** [vuematerial/vue-material#931]
Could you guys show me the right way to use md-stepper?

### Steps to reproduce
```
<template>
  <div class="app-container">
    <md-stepper md-vertical :md-elevation="step">
          <md-step md-label="step1"><p>step 1</p></md-step>
          <md-step md-label="step2"><p>step 2</p></md-step>
          <md-step md-label="step3"><p>step 3</p></md-step>
    </md-stepper>
  </div>
</template>

<script>
export default {
  data() {
    return {
      step: 1
    };
  },

  mounted() {
    this.step = 2;
  }
}
</script>
```
### Which browser?

Chrome 59.0.3071.115
### What is expected?
the stepper goes to step 2
### What is actually happening?

still on step 1

**Comment by vitorcominato** [comment]
The prop "md-elevation" is to set the elevation of the container for each content and the horizontal header.
Is the effect of elevation on the container.

**Comment by zapping** [comment]
@vitorcominato  Have a look at this pen, https://codepen.io/zapping/pen/xLOjBp . This should get you started.

**Comment by ansonhorse** [comment]
@vitorcominato Thanks!
@zapping Thanks!  And I'm wonder how can I find the methods of this component? It's not listed on the documentation, [http://vuematerial.io/#/components/stepper](url)

**Comment by zapping** [comment]
@ansonhorse  Actually its been resolved in the issue #905. Its reached to the dev branch and its yet to be pulled to the master.

**Comment by githrdw** [comment]
@ansonhorse I think your question has not been answered yet?
If I understand your question correctly, you want to change the step number with a function.
I was also looking for it and found a workaround:

1. Give your stepper a reference by giving the `ref`-tag
2. Add the following code to the `mounted()` function:
```
var stepper = this.$refs['YOUR_STEPPER_REFERENCE'];
var stepNumber = this.stepNumber;
var stepKey = Object.keys(stepper.stepList)[stepNumber];
var stepData = stepper.stepList[stepKey];
stepper.setActiveStep(stepData);
```
Or use the minified way:

```
var stepper = this.$refs['YOUR_STEPPER_REFERENCE'];
stepper.setActiveStep(stepper.stepList[Object.keys(stepper.stepList)[this.stepNumber]])
```

@pablohpsilva Maybe it's possible to add a built-in function so you can do something like `this.$refs['stepper'].setStepNumber(3)`?
